src/npmUtils.ts

```diff
@@ -0,0 +1,59 @@
+import fs from "fs";
+import * as ini from "ini";
+
+// https://docs.npmjs.com/using-private-packages-in-a-ci-cd-workflow#create-and-check-in-a-project-specific-npmrc-file
+const npmRegistryTokenKey = "//registry.npmjs.org/:_authToken";
```

Comment:
I did some research and couldn't find a case where the value isn't exactly like this (for the npm registry). Therefore I think we can omit the regex and simply do a 1:1 check.